accord-core/src/main/java/accord/primitives/Deps.java

```diff
@@ -1022,15 +1049,27 @@ public TxnId txnId(int i)
         return txnIds[i];
     }
 
+    @VisibleForImplementation
+    public int keyToTxnIdCount()
+    {
+        return keyToTxnId.length;
+    }
+
+    @VisibleForImplementation
+    public int keyToTxnId(int idx)
```

Comment:
Perhaps prefix with `unsafe` or `serialization` as this is only for serialisation, and is not to be interpreted directly? As named it might be misinterpreted as a function for external use, which would be unlikely to go well.